src/dashboard/Data/Config/ConfigDialog.react.js

```diff
@@ -228,6 +232,21 @@ export default class ConfigDialog extends React.Component {
               description='Use this to configure your app. You can change it at any time.' />
           }
           input={EDITORS[this.state.type](this.state.value, (value) => { this.setState({ value }) })} />
+        <Field
+          label={
+            <Label
+              text='Requires master key?'
+              description='The parameter is returned only when requested with the master key. You can change it at any time.' />
```

Comment:
Probably good to be super explicit...
```suggestion
              description='When true the parameter is returned only when requested with the master key. You can change it at any time.' />
```

Reply:
As @davimacedo suggested, we are going away from the notion of “true/false” in favor of “yes/no”. Do you want to change the wording in your suggestion?

Reply:
As @davimacedo suggested, we are going away from the notion of “true/false” in favor of “yes/no”. Do you want to change the wording in your suggestion?

Reply:
As @davimacedo suggested, we are going away from the notion of “true/false” in favor of “yes/no”. Do you want to change the wording in your suggestion?

Reply:
Sure